docs/sources/developers/plugins/build-a-streaming-data-source-plugin.md

```diff
@@ -95,9 +106,9 @@ Grafana uses [RxJS](https://rxjs.dev/) to continuously send data from a data sou
    };
    ```
 
-   > **Note:** In practice, you'd call `subscriber.next` as soon as you receive new data from a websocket or a message bus. The example above simulates data being received every 500 milliseconds.
+   > **Note:** In practice, you'd call `subscriber.next` as soon as you receive new data from a websocket or a message bus. In the example above, data is being received every 500 milliseconds.
```

Comment:
I found it strange that we're showing this... what do folks think of instead linking to and referencing our sample https://github.com/grafana/grafana-plugin-examples/tree/main/examples/datasource-streaming-websocket?

Reply:
my 2 cents: I think we want at least one good, detailed example in each how-to/tutorial in Create a plugin; and we should add links to relevant samples in grafana-plugin-examples. That said, if this isn't a strong example to show for whatever reason, let's remove it until a better example is created.

Reply:
I'd like to postpone deleting the example until we get to phase 3.

Reply:
I have created https://github.com/grafana/grafana/issues/67566 to track.